content/en/tracing/profiling/_index.md

```diff
@@ -181,11 +190,25 @@ Profiler configuration:
 |  WithAPIKey      | String        | The Datadog [Datadog API key][2]                                                                             |
-|  WithService     | String        | The Datadog [service][4] name, for example `my-web-app`, which can be set here, or in `DD_TAGS`.             |
-|  WithEnv         | String        | The Datadog [environment][5] name, for example `production`, which can be set here, or in `DD_TAGS`.         |
-|  WithTags        | String        | The tags to apply to an uploaded profile. Must be a list of in the format `<KEY1>:<VALUE1>,<KEY2>:<VALUE2>`. |
+|  WithService     | String        | The Datadog [service][4] name, for example `my-web-app`, which can be set here, or in `DD_SERVICE`.             |
+|  WithEnv         | String        | The Datadog [environment][5] name, for example `production`, which can be set here, or in `DD_ENV`.         |
+|  WithTags        | String        | The tags to apply to an uploaded profile. Must be a list of in the format `<KEY1>:<VALUE1>,<KEY2>:<VALUE2>`. You can also set this in `DD_TAGS`. |
+|  WithSite        | String        | If your organization is on Datadog EU site, set this to `datadoghq.eu`. You can also set this in `DD_SITE`. |
 
 
+- For advanced setup of the profiler or to add tags like `service` or `version`, use environment variables to set the parameters:
+
+| Environment variable                             | Type          | Description                                                                                      |
+| ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------ |
+| `DD_PROFILING_ENABLED`                           | Boolean       | Alternate for `-Ddd.profiling.enabled` argument. Set to `true` to enable profiling.               |
+| `DD_API_KEY`                                     | String        | Alternate for `-Ddd.profiling.api-key-file` argument. The [Datadog API key][2] to use when uploading profiles. |
+| `DD_PROFILING_API_KEY_FILE`                      | String        | Replacement for `-Ddd.profiling.api-key-file` argument. File that should contain the [Datadog API key][2].  |
+| `DD_SITE`                                        | String        | If your organization is on Datadog EU site, set this to `datadoghq.eu`.                          |
+| `DD_SERVICE`                                     | String        | The Datadog [service][4] name.     |
+| `DD_ENV`                                         | String        | The Datadog [environment][5] name, for example `production`.|
+| `DD_VERSION`                                     | String        | The version of your application.                              |
+| `DD_TAGS`                                        | String        | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` separated by commas such as: `layer:api, team:intake`.  |
```

Comment:
Is this section a mistake?

Reply:
Looks like we do support a few of these env variables but I just realized that I forgot to remove some 😢 . Will update!